app/components/emoji_picker/emoji_picker.js

```diff
@@ -69,6 +69,10 @@ class EmojiPicker extends PureComponent {
         });
     }
 
+    filterEmojiAliases = (aliases, searchTerm) => {
+        return aliases.findIndex((alias) => alias.includes(searchTerm)) !== -1;
```

Comment:
Yeah, it does. `findIndex` returns the index of the first item where its argument evaluates true though, so that's what we want